### DISCUSSION TITLE
Open Script Button In ECS View

### DISCUSSION THREAD

**Discussion by TCROC** [GodotECS/godex#229] [Ideas & Proposals]
It would be nice to have the ability to open the the System / Component script from the ECS View.  This would nicer than having to find the script in the project to edit it.  A possible UI design could be a button just to the left of the script name and invoke the same logic that the "Open" context menu option invokes.  It looks like the "Open" context menu option already has an icon we can use for the button.

![image](https://user-images.githubusercontent.com/24307049/116273032-698c7600-a74f-11eb-9d59-1f29871b24a7.png)

**Comment by AndreaCatania** [comment]
This is a great idea!

**Comment by AndreaCatania** [comment]
Moving the proposal to discussions to keep the issue just for things to fix.
